app/templates/components/layer-record-views/tax-lot.hbs

```diff
@@ -36,6 +36,10 @@
 {{#if this.model.zonedist1}}
   <ul class="lot-zoning-list">
     <li class="menu-text">
+      {{!-- WORK WITH THIS FILE FOR THE ADDING NEW SPECIAL PURPOSE DISTRICTS --}}
```

Comment:
Maybe we can remove this comment or at least change it to provide a little more context? This is just a template that happens to do some stuff with SPDs. There are plenty of other files that reference or use them as well.

Reply:
woops this was just for me. This PR isn't 100% ready for a code review actually.